templates/register/form.html

```diff
@@ -0,0 +1,23 @@
+<div class="flex h-screen items-center justify-center">
+    <div class="w-96 rounded bg-white p-8 shadow-md">
+        <h1 class="mb-4 text-2xl font-bold">REGISTER</h1>
+        <form hx-post="register" hx-target="body">
+            {{ username|safe }}
+            <div class="flex flex-row gap-x-1">
+                <!-- prettier-ignore -->
+                {{ first_name|safe }} 
+                {{ last_name|safe }}
+            </div>
+            <!-- prettier-ignore -->
+            {{ email|safe }}
+            {{ password|safe }}
+            {{ confirm_password|safe }}
```

Comment:
Co to znamená to "safe"?

Reply:
Askama defaultně escapuje HTML znaky, takže z ">" udělá "&gt" a tak dále. Safe znamená safe to render, je to filtr co vypíná to escapovaní. Pokud to tam nedáš tak ti to místo elementů vyblije na stránku hromadu nesmyslnýho textu.